Sources/MuxPlayerSwift/GlobalLifecycle/PlayerSDK.swift

```diff
@@ -15,8 +16,38 @@ class PlayerSDK {
 
     let monitor: Monitor
 
+    let keyValueObservation: KeyValueObservation
+
     init() {
         self.monitor = Monitor()
+        self.keyValueObservation = KeyValueObservation()
+    }
+
+    class KeyValueObservation {
+
+        var observations: [ObjectIdentifier: NSKeyValueObservation] = [:]
+
+        func register<Value>(
+            _ player: AVPlayer,
+            for keyPath: KeyPath<AVPlayer, Value>,
+            options: NSKeyValueObservingOptions,
+            changeHandler: @escaping (AVPlayer, NSKeyValueObservedChange<Value>) -> Void
+        ) {
+            let observation = player.observe(keyPath,
+                                             options: options,
+                                             changeHandler: changeHandler
+            )
```

Comment:
question: Does this allow for multiple `NSKeyValueObservation`s per player? (ie, for different key paths)